### DISCUSSION TITLE
希望能够慢慢实现的功能

### DISCUSSION THREAD

**Discussion by bilibili2233a** [ReadYouApp/ReadYou#51] [Ideas]
图1：希望能够将不允许通知的rss源不加入到全部未读统计。未读通知太多了，点进去全是不允许通知的和不重要的。
![Screenshot_20220506_235414_me.ash.reader.jpg](https://user-images.githubusercontent.com/99982054/167168786-d6a5e4a2-b919-4cce-b12b-468acc7927be.jpg)
图2：rss源一多每次冷启动进入同步速度太慢了，感觉不如可以选择不重要的rss源在指定的时间段（睌上凌晨空闲、几小时、天、周）后台自启动同步一次!重要的rss源可以选择（半小时、几小时、凌晨空闲&中午空闲）后台自启动。[Screenshot_20220506_235947_me.ash.reader.jpg](https://user-images.githubusercontent.com/99982054/167170510-f726d677-f803-4546-9f20-c6dffc561b85.jpg)
3.图片点击缩放自由旋转功能：遇到需要的时候，往往只能访问原网页才能缩放。

以下为可以不实现的功能：
4.文字批注功能（用处需求不大，可以不实现）
5.右上角的二级菜单功能：
1.分享原网页链接功能（调用系统级分享菜单，参考ui）![Screenshot_20220507_002126_com.huawei.android.internal.app.jpg](https://user-images.githubusercontent.com/99982054/167172901-b2cb844b-f92a-4407-992e-036e0967f1f4.jpg)
2.复制全文（省去了全选时不小心多复制的可能）
3.窗口标签页进入（在多个之间自由切换，参考例子：chrome浏览器的）![Screenshot_20220507_002629_com.android.chrome.jpg](https://user-images.githubusercontent.com/99982054/167173723-d875239d-a437-441e-bdb1-0a5eefbd4384.jpg)

**Comment by Ashinch** [comment]
分享和同步频率设置是在计划中的，不统计关闭通知的文章数量是个不错的点子，应该可以弄个开关来自定义。针对大量订阅源的优化估计要到后面一些，很想去做，但是需要花很多时间去一步步测试和调试。
